core/templates/dev/head/components/question-directives/question-player/question-player.constants.ts

```diff
@@ -27,18 +30,8 @@ angular.module('oppia').constant(
     // color shallow green
     PASSED_COLOR_OUTER: 'rgb(143, 217, 209)'
   }
-);
-
-angular.module('oppia').constant('HASH_PARAM', 'question-player-result=');
-
-angular.module('oppia').constant('MAX_SCORE_PER_QUESTION', 1.0);
 
-angular.module('oppia').constant(
-  'QUESTION_PLAYER_MODE', {
+  public static QUESTION_PLAYER_MODE = {
     PASS_FAIL_MODE: 'PASS_FAIL'
   }
-);
-
-angular.module('oppia').constant('VIEW_HINT_PENALTY', 0.1);
```

Comment:
Where did these constants go to? /cc @sophiewu6 @vinitamurthi

Reply:
Restored them!

Reply:
Could you please check and verify that nothing else is missing?

Also I suggest you sort out #7292 in this PR, since you've just pushed it.

Reply:
Done!